python/Ganga/Utility/execute.py

```diff
@@ -207,7 +207,7 @@ def execute(command,
     else:
         # We want to run a shell command inside a _NEW_ shell environment.
         # i.e. What I run here I expect to behave in the same way from the command line after I exit Ganga
-        stream_command = "bash -i "
+        stream_command = "bash "
```

Comment:
Because previously when sourcing the environment, it was giving so many errors that an interactive bash session was failing to complete. Furthermore, we are not running interactively so it's only likely so confuse scripts which check for how they are running and then try to request input from the user.